hack/cr.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
   name: "elasticsearch"
   annotations:
       elasticsearch.openshift.io/loglevel: trace
+      logging.openshift.io/elasticsearch-cert-management: "true"
+      logging.openshift.io/elasticsearch-cert.fluentd: "system.logging.fluentd"
```

Comment:
Enhancement proposal states the secret name is delimited using a slash instead of a period

Reply:
needed to update proposal because k8s doesn't allow more than one `/`